SpatialGDK/Source/Public/Interop/SnapshotManager.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/NoExportTypes.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Utils/SchemaUtils.h"
+
+#include <improbable/c_schema.h>
+#include <improbable/c_worker.h>
+
+#include "SnapshotManager.generated.h"
+
+class UGlobalStateManager;
```

Comment:
Can you forward declare the NetDriver and SpatialReceiver too?

Reply:
We need the NetDriver due to our use of const `USpatialNetDriver::ServerTravelDelegate& Delegate` here